app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) == 3:
+        action, source_file, destination_file = parts
+
+        if action == "cp" and source_file != destination_file:
+            try:
+                with open(source_file, "r") as file_in:
+                    with open(destination_file, "w") as file_out:
```

Comment:
You can actually write it in one line! Let's do this last fix 💪 Looks good overall 😌

Reply:
line to long